build.gradle

```diff
@@ -15,10 +15,6 @@ subprojects { Project subproject ->
 
     apply plugin: "io.micronaut.build.common"
 
-    if (subproject.parent.name == "examples" || subproject.name == "examples") {
```

Comment:
why did you remove this?

Reply:
We've fixed that in the build plugin (in version 2.0.10), so that's not necessary anymore. See https://github.com/micronaut-projects/micronaut-build/commit/07b957785beb237b5ed7a728d12b0b9262380987

Reply:
We've also updated the project template so new projects don't need to use have it.